examples/config files - basic/1 user-sync-config.yml

```diff
@@ -212,11 +212,16 @@ logging:
 
   # (optional) file_log_directory (default value "logs")
   # An absolute or relative path to the directory where log files should be placed.
-  # A single file is created per day, named with format "YYYY-MM-DD.log", and all
-  # logging for runs started on that day (local time) are placed in that file.
   # If a relative path is specified, it is interpreted relative to this configuration
   # file, not relative to the User Sync process working directory.
   file_log_directory: logs
+  
+  # (optional) file_log_name (default value "default")
```

Comment:
The name of this parameter  should be `file_log_name_format`.

The value of this parameter should be a Python format string that can refer to a single positional argument which is a `datetime.datetime` object produced by calling `datetime.now()` at the start of the run.

The default value of this string should be `'{%Y-%m-%d}.log'`, which produces the current default value.